src/workers/postgres.rs

```diff
@@ -162,15 +170,20 @@ impl Postgres {
             quic_response,
         } = tx;
 
-        self.client.read().await.execute(
-            r#"
-                INSERT INTO lite_rpc.Txs 
-                (signature, recent_slot, forwarded_slot, processed_slot, cu_consumed, cu_requested, quic_response)
-                VALUES
-                ($1, $2, $3, $4, $5, $6, $7)
-            "#,
-            &[&signature, &recent_slot, &forwarded_slot, &processed_slot, &cu_consumed, &cu_requested, &quic_response],
-        ).await?;
+        self.client
```

Comment:
How is batching done ?

Reply:
So that is taken care of internally by the library. U just have to pre prepare the client with the queries

Reply:
There is no way to send multiple transactions at a time ?

Reply:
yes there is https://www.digitalocean.com/community/tutorials/sql-insert-multiple-rows it will dramatically speed up this code